sharkfuser/include/fusilli/attributes/tensor_attributes.h

```diff
@@ -257,6 +257,62 @@ getContiguousToChannelsLastPermuteOrder(size_t numDims) {
   return permuteOrder;
 }
 
+// Given a permutation vector, returns its inverse.
+inline std::vector<int64_t>
+inversePermutation(const std::vector<int64_t> &permutation) {
+  std::vector<int64_t> inverse(permutation.size());
+  for (size_t i = 0; i < permutation.size(); ++i) {
+    inverse[permutation[i]] = static_cast<int64_t>(i);
+  }
+  return inverse;
+}
+
+// This computes the permutation needed to go from logical dims to physical dims
+// based on the dims and stride. Unit length dimensions are kept in their
+// original positions as they don't meaningfully contribute to the layout.
+inline std::vector<int64_t>
+getLogicalToPhysicalPerm(const std::vector<int64_t> &dim,
```

Comment:
```suggestion
getLogicalToPhysicalPermuteOrder(const std::vector<int64_t> &dim,
```
for consistency with similar utilities that return the permute order